.evergreen/serverless/create-instance.sh

```diff
@@ -9,6 +9,17 @@ if [ -z "$PROJECT" ]; then
 fi
 INSTANCE_NAME="$RANDOM-$PROJECT"
 
+# Use the LOADBALANCED environment variable to opt-in to testing
```

Comment:
If this approach looks good, I can create a DRIVERS ticket as follow up work after DRIVERS-1603 is implemented in language teams to remove the `LOADBALANCED` opt-in environment variable, and always use `AWS` as the backing provider.